pyanaconda/display.py

```diff
@@ -328,8 +326,10 @@ def setup_display(anaconda, options):
             log.info("VNC requested via kickstart, switching Anaconda to GUI mode.")
             anaconda.display_mode = constants.DisplayModes.GUI
 
+        log.debug(vnc_server.password)
         if vnc_server.password == "":
-            vnc_server.password = vnc_data.password
+            log.debug(vnc_data.password.value)
```

Comment:
I dont think we should print passwords :)

Reply:
Sure - still with VNC (no encryption & very short maximum password length) it would be less of an issue. ;-)